tests/sles4sap/publiccloud/clean_leftover_peerings.pm

```diff
@@ -0,0 +1,31 @@
+# Copyright SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+# Summary: Deployment steps for qe-sap-deployment
+# Maintainer: QE-SAP <qe-sap@suse.de>
+
+use strict;
+use warnings;
+use base 'sles4sap_publiccloud_basetest';
+use testapi;
+use qesapdeployment;
+use publiccloud::utils qw(is_azure is_ec2);
+
+sub test_flags {
+    return {fatal => 1, publiccloud_multi_module => 1};
+}
+
+sub run {
+    my ($self, $run_args) = @_;
+    if (is_azure) {
+        if (!get_var('IBSM_RG')) {
```

Comment:
silently exit? Why not `get_required_var` and die?

Reply:
To not fail in a hypothetical case where this will be used in a context without `IBSM_RG` (for example, if in the future we decide to create a job only cleaning old peerings). If the `IBSM_RG` var is missing in an actual IBSM job, it will fail in other places clearly anyway.